src/platform_impl/macos/view.rs

```diff
@@ -273,9 +273,10 @@ declare_class!(
         fn set_marked_text(
             &self,
             string: &NSObject,
-            _selected_range: NSRange,
+            selected_range: NSRange,
             _replacement_range: NSRange,
         ) {
+            // TODO: Use `_replacement_range` correctly (known to be used by Japanese IMEs).
```

Comment:
Same comment as below for your _bigger_ todo, I'd suggest to just keep one of them present, likely right above the `_replacement_range`